index.html

```diff
@@ -97,7 +97,18 @@
     </section>
 
     <section id="sotd">
-    
+      <p class="advisement">
+        This work item has been 
+        <a href="https://www.w3.org/2021/Process-20211102/#discontinuedREC">discontinued</a>. 
```

Comment:
```suggestion
        <a href="https://www.w3.org/2023/Process-20230612/#discontinuedREC">discontinued</a>. 
```

(The new process has just been published... ☹️)